pkg/controllers/resourcechange/resourcechange_controller_test.go

```diff
@@ -1930,9 +2443,8 @@ func TestTriggerAffectedPlacementsForUpdatedClusterRes(t *testing.T) {
 				ResourcePlacementController: &fakeController{QueueObj: []string{}},
 			}
 
-			result, err := reconciler.triggerAffectedPlacementsForUpdatedClusterRes(tt.key, tt.resource, tt.key.Namespace == "")
-			if err != nil || result.Requeue || result.RequeueAfter > 0 {
-				t.Fatalf("triggerAffectedPlacementsForUpdatedClusterRes = %v, %v, want empty result and nil err", result, err)
+			if err := reconciler.triggerAffectedPlacementsForUpdatedRes(tt.key, tt.resource, tt.key.Namespace == ""); err != nil {
```

Comment:
When deleting a namespaced resource, `triggerAffectedPlacementsForUpdateRes` is called with a namespaced key (original resource), nil resource, and isClusterScope set to true. Is this combination tested?